daemon/Makefile

```diff
@@ -1,233 +0,0 @@
-
-VERSION   ?= $(shell awk '/^Version:/ {print $$2}' ../rpm/sailjail.spec)
-
-# ----------------------------------------------------------------------------
-# Installation directories and pkg-config
-# ----------------------------------------------------------------------------
-
-include Makefile.inc
-
-# ----------------------------------------------------------------------------
-# List of targets to build
-# ----------------------------------------------------------------------------
-
-TARGETS_BIN += sailjaild
-TARGETS_BIN += sailjail
-
-## QUARANTINE TARGETS_PNG += sailjaild.png
-
-TARGETS += ${TARGETS_BIN} ${TARGETS_PNG}
-
-# ----------------------------------------------------------------------------
-# Top level targets
-# ----------------------------------------------------------------------------
-
-.PHONY: build install clean distclean mostlyclean test-build test-install test-clean
-
-build:: $(TARGETS)
-
-install:: build
-	install -d ${DESTDIR}${_BINDIR}
-	install -m755 ${TARGETS_BIN} ${DESTDIR}${_BINDIR}
-
-clean:: mostlyclean
-	$(RM) $(TARGETS)
-
-distclean:: clean
-	$(RM) tags
-
-mostlyclean::
-	$(RM) *.o *~ *.bak */*.o */*~ */*.bak
-
-test-build::
-	$(MAKE) -C test build
-
-test-install::
-	$(MAKE) -C test install
-
-test-clean::
-	$(MAKE) -C test clean
-
-# ----------------------------------------------------------------------------
-# Implicit rules
-# ----------------------------------------------------------------------------
-
-%.so : %.o
-	$(CC) -o $@ -shared $^ $(LDFLAGS) $(LDLIBS)
-
-%.png : %.dot
-	dot -Tpng $^ -o $@
-
-# ----------------------------------------------------------------------------
-# Explicit dependencies
-# ----------------------------------------------------------------------------
-
-# - - - - - - - - - - - - - - - - - - - -
-# sailjaild
-# - - - - - - - - - - - - - - - - - - - -
-
-sailjaild_objs += sailjaild.o
-
-sailjaild_objs += appinfo.o
-sailjaild_objs += applications.o
-sailjaild_objs += config.o
-sailjaild_objs += control.o
-sailjaild_objs += later.o
-sailjaild_objs += logging.o
-sailjaild_objs += mainloop.o
-sailjaild_objs += migrator.o
-sailjaild_objs += permissions.o
-sailjaild_objs += prompter.o
-sailjaild_objs += service.o
-sailjaild_objs += session.o
-sailjaild_objs += settings.o
-sailjaild_objs += stringset.o
-sailjaild_objs += users.o
-sailjaild_objs += util.o
-
-sailjaild : ${sailjaild_objs}
-
-# - - - - - - - - - - - - - - - - - - - -
-# sailjailclient
-# - - - - - - - - - - - - - - - - - - - -
-
-sailjailclient : sailjailclient.o util.o logging.o util.o stringset.o config.o
-
-clean::
-	$(RM) sailjailclient
-
-# - - - - - - - - - - - - - - - - - - - -
-# sailjail
-# - - - - - - - - - - - - - - - - - - - -
-
-sailjail : sailjailclient
-	cp $< $@
-
-# - - - - - - - - - - - - - - - - - - - -
-# desktop_data
-# - - - - - - - - - - - - - - - - - - - -
-
-desktop_data : desktop_data.o util.o logging.o stringset.o
```

Comment:
I forgot this little tool from meson.build.

Reply:
Added it there. It's built with `meson compile -C build desktop_data` but not by default.